README.rst

```diff
@@ -140,60 +177,54 @@ If you want to build "final" Docker images, then use:
     # GPU
     docker build -t preprod-mxnet:0.12.1-gpu-py2 -f Dockerfile.gpu .
 
-    # For building images of MXNet versions 1.1 and above
-    docker build -t preprod-mxnet:1.1.0-cpu-py2 --build-arg py_version=2
-    --build-arg framework_installable=mxnet-1.1.0-py2.py3-none-manylinux1_x86_64.whl -f Dockerfile.cpu .
-
 
 Running the tests
 -----------------
 
-Running the tests requires installation of the SageMaker MXNet Container code and its test
-dependencies.
+Running the tests requires installation of the SageMaker MXNet Container code and its test dependencies.
 
 ::
 
-    git clone https://github.com/aws/sagemaker-mxnet-containers.git
-    cd sagemaker-mxnet-containers
+    git clone https://github.com/aws/sagemaker-mxnet-container.git
+    cd sagemaker-mxnet-container
     pip install -e .[test]
 
-Tests are defined in
-`test/ <https://github.com/aws/sagemaker-mxnet-containers/tree/master/test>`__
-and include unit, integration and functional tests.
+Tests are defined in `test/ <https://github.com/aws/sagemaker-mxnet-containers/tree/master/test>`__ and include unit and integration tests.
+The integration tests include both running the Docker containers locally and running them on SageMaker.
+The tests are compatible with only the Docker images built by Dockerfiles in the current branch.
+If you want to run tests for MXNet versions 1.2.1 or below, please use the v1.0.0 tests.
+
+All test instructions should be run from the top level directory
 
 Unit Tests
 ~~~~~~~~~~
 
-If you want to run unit tests, then use:
+To run unit tests:
 
 ::
 
-    # All test instructions should be run from the top level directory
-
     pytest test/unit
 
 Local Integration Tests
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-Running local integration tests require `Docker <https://www.docker.com/>`__ and `AWS
-credentials <https://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/setup-credentials.html>`__,
-as the integration tests make calls to a couple AWS services. All integration
-tests require configurations specified within their respective
-`conftest.py <https://github.com/aws/sagemaker-mxnet-containers/blob/master/test/conftest.py>`__.
-
-Local integration tests on GPU require `Nvidia-Docker <https://github.com/NVIDIA/nvidia-docker>`__.
+Running local integration tests require `Docker <https://www.docker.com/>`__ and `AWS credentials <https://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/setup-credentials.html>`__,
+as the integration tests make calls to a couple AWS services.
+Local integration tests on GPU require `nvidia-docker2 <https://github.com/NVIDIA/nvidia-docker>`__.
+You Docker image must also be built in order to run the tests against it.
 
-Before running local integration tests:
+Local integration tests use the following pytest arguments:
```

Comment:
Should we mention the gpu tests need nvidia docker here?

Reply:
it's on line 213 already.